public/introduce.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8"/>
+    <link rel="stylesheet" href="introduce.css">
```

Comment:
```suggestion
    <link rel="stylesheet" href="introduce.css" />
```
タグを閉じた方が良いかもです！（その後の要素のインデントがずれてしまっているので）